src/functions.php

```diff
@@ -467,17 +467,15 @@ function exportPhonebook(array $cards, array $conversions): SimpleXMLElement
  * @param array             $config
  * @return void
  */
-function uploadPhonebook(SimpleXMLElement $xmlPhonebook, array $config)
```

Comment:
I would prefer the params to be fritzConfig and phonebookConfig to make clear what those are?